app/views/staff/organisations/_grouped_programmes.html.haml

```diff
@@ -25,6 +25,6 @@
                   %td{class: "govuk-table__cell"}
                     = activity.roda_identifier_compound
                   %td{class: "govuk-table__cell"}
-                    - if policy(:project).create?
+                    - if policy(:project).create? && policy(activity).create_child?
```

Comment:
I think, although I'm not 100%, but based on the scope of this card https://trello.com/c/b7wCEUkE/1644-use-createchild-when-deciding-whether-to-show-add-activity-buttons , that the `policy(:project).create?` check is now redundant and can be removed.

Reply:
Yeah, I think you're right. I'll see what happens if I get rid